src/rpc/snapshots/.gitattributes

```diff
@@ -0,0 +1 @@
+*.snap linguist-generated=true
```

Comment:
This is not a well-known feature; let's add a link/comment to help readers understand it.